packages/snaps-execution-environments/src/common/globalObject.ts

```diff
@@ -9,6 +9,14 @@ enum GlobalObjectNames {
   window = 'window',
   /* eslint-enable @typescript-eslint/naming-convention */
 }
+declare global {
```

Comment:
Can we move this to type only file like https://github.com/MetaMask/snaps/blob/main/packages/snaps-controllers/src/types/vendor/global.d.ts? We may need to create a new one in `snaps-execution-environments`

Reply:
created a more generic interface to address this but will iterate further trying to create something less open.